app/models/copilot/content.rb

```diff
@@ -0,0 +1,11 @@
+module Copilot
+  class Content < ActiveRecord::Base
+    validates :slug, presence: true, uniqueness: true
+
+    def self.fetch(slug, value='')
+      content = self.find_or_create_by(slug: slug)
+      content.update(value: value) unless content.value.present?
```

Comment:
So basically you only want to update the content's value if it currently is blank? Is this to set a default value? You should be able write as:

```ruby
content = self.find_or_create_by(slug: slug) do |content|
  content.value = value
end
```